core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -1154,6 +1154,11 @@ public function generateAlias($varValue, DataContainer $dc)
 			}
 		}
 
+		if ($autoAlias === false && $varValue !== $dc->activeRecord->alias)
```

Comment:
I don’t think `$autoAlias` has to be checked here. If someone empties the alias field the search index should still be purged.

```suggestion
		if ($varValue !== $dc->activeRecord->alias)
```

Reply:
You're right, that check did more than what I intended it to do. The intent was to not purge the index if there was no alias due to the page just being created. If the alias is created for the very first time during page creation, there is no need for a purge attempt.

I'm now using `Input::post('VERSION_NUMBER') > 0` to check for that.